argo/workflow/pipelined-pessimistic-locking.yaml

```diff
@@ -0,0 +1,62 @@
+metadata:
+  generateName: tipocket-pipelined-pessimistic-locking-
+  namespace: argo
+spec:
+  entrypoint: call-tipocket-pipelined-pessimistic-locking
+  arguments:
+    parameters:
+      - name: ns
+        value: tipocket-pipelined-pessimistic-locking
+      - name: purge
+        value: "true"
+      - name: image-version
+        value: release-4.0-nightly
+      - name: storage-class
+        value: local-storage
+      - name: nemesis
+        value: ""
```

Comment:
how about enable `shuffle region scheduler` and `random kill`?

Reply:
This test focuses on the success rate of pipelined locking in a normal environment. If enables `random kill`, some transaction can fail and we can't know whether it's caused by pipelined locking or node failure.

Reply:
I think `shuffle region scheduler` is covered by `scatter table`?